robottelo/cli/factory.py

```diff
@@ -1693,12 +1693,17 @@ def setup_org_for_a_custom_repo(options=None):
     # Get the content view info
     cv_info = ContentView.info({'id': cv_id})
     lce_promoted = cv_info['lifecycle-environments']
-    cvv = cv_info['versions'][-1]
+    assert len(cv_info['versions']) > 0
+    cvv = sorted(cvv['id'] for cvv in cv_info['versions'])[-1]
```

Comment:
Non-blocking, Sorting is fine here, but why we're explicitly hitting this issue in stream(seen a couple of PRs for this), so just wondering if are you aware of any related changes in stream? @damoore044 @vsedmik @sambible

Reply:
@Gauravtalreja1  The issue is not specific to stream, I've seen it in older versions before and [this BZ](https://bugzilla.redhat.com/show_bug.cgi?id=2209758) was reported already. However, it feels to happen more frequently in stream - same passes in z-streams but fails in stream.